packages/mui-material-next/src/IconButton/IconButton.test.js

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import PropTypes from 'prop-types';
+import { createRenderer, describeConformance } from '@mui-internal/test-utils';
+import { unstable_capitalize as capitalize } from '@mui/utils';
+import Icon from '@mui/material/Icon';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import ButtonBase from '@mui/material/ButtonBase';
+import { iconButtonClasses as classes } from '@mui/material/IconButton';
```

Comment:
Importing these from `material-next` right now now will fail most of the tests because of `focusRipple` being dropped from `ButtonBase`, easier to keep them as-is instead of adding `.skip()` to a bunch of tests